### PR TITLE
Basic case study listing and individual view

### PR DESCRIPTION
**Pull request by maxious**
No editing/deleting and supplier view isn't entirely react-ified but good point to merge back.

### CODE REVIEW

File: app/api_client/data.py [hunk 1]
```diff
@@ -15,6 +15,11 @@ def get_supplier(self, supplier_code):
             "/suppliers/{}".format(supplier_code)
         )
 
+    def get_case_study(self, case_study_id):
```
Comment (matt-sm) [inline app/api_client/data.py]: I dont remember updating this file for other api client changes...

File: app/helpers/shared_helpers.py [hunk 2]
```diff
@@ -12,3 +15,10 @@ def get_label_for_lot_param(lot_param):
 
 def chunk_string(string, chunk_length):
     return (string[0+i:chunk_length+i] for i in range(0, len(string), chunk_length))
+
+
+def request_wants_json():
```
Comment (matt-sm) [inline app/helpers/shared_helpers.py]: http://flask.pocoo.org/snippets/45/ by any chance :)

File: app/templates/suppliers_details.html [hunk 3]
```diff
@@ -55,14 +55,26 @@
...
         {% endfor %}
       </footer>
     </article>
   </li>
+  {% if supplier["caseStudies"]|count > 0 %}
+  <li>
+    <p>Case Studies</p>
+    <article>
+      {% for casestudy in supplier["caseStudies"] %}
+        {% with component = render_component('bundles/CaseStudy/CaseStudyViewWidget.js',{ "casestudy": casestudy }) %}
```
Comment (matt-sm) [inline app/templates/suppliers_details.html]: won't this display the full case study?

don't we just want a list of case studies with titles that links to the view page?

(I would just do in jinja for now)

File: app/api_client/data.py (hunk 1 above)
Comment (maxious) [inline app/api_client/data.py]: Hmm this folder seems to be a subset of the real apiclient code

File: app/templates/suppliers_details.html (hunk 3 above)
Comment (maxious) [inline app/templates/suppliers_details.html]: Added jinja previews that link to the full view to match designs.

**Pull request merged**